### ISSUE TITLE
The int-5-mano-plugin-management job is failling after PR#192

### ISSUE THREAD

**Issue by felipevicens** [sonata-nfv/son-mano-framework#194]
@tsoenen Last PR produced this error in the integration tests. Could you have a look?

https://jenkins.sonata-nfv.eu/job/int-5-mano-plugin-management/188/console

```
03:00:19 + python test.py http://sp.int3.sonata-nfv.eu:8001
03:00:19 ================================================
03:00:19 BEGIN test.py
03:00:19 [u'e5e8d1c5-a64b-4bb9-89f5-838b58e14ca1']
03:00:19 {u'uuid': u'e5e8d1c5-a64b-4bb9-89f5-838b58e14ca1', u'last_heartbeat_at': u'None', u'registered_at': u'2017-03-18 22:13:49.134000', u'name': u'son-plugin.TestPlugin', u'state': u'REGISTERED', u'version': u'1.0-dev', u'description': None}
03:00:19 Traceback (most recent call last):
03:00:19   File "test.py", line 57, in <module>
03:00:19     main()
03:00:19   File "test.py", line 41, in main
03:00:19     raise BaseException("Test plugin state != RUNNING")
03:00:19 BaseException: Test plugin state != RUNNING
03:00:19 Build step 'Execute shell' marked build as failure
```

**Comment by tsoenen** [comment]
@felipevicens @mpeuster : I investigated the issue, but I have no idea what is causing it. If I redeploy the environment, and I redo the test, it succeeds. If I redeploy the environment, and redo the chain of integration tests as they are performed automatically at night, it succeeds. 

I noticed that, although the test succeeded before my commits, they were not performed correctly. For example, look at https://jenkins.sonata-nfv.eu/job/int-5-mano-plugin-management/181/console. Although this test is successful, the list that is shown between BEGIN clean.py and END clean.py is empty, while it is supposed to show all the plugins connected to the plugin manager (SLM, SMR, ...). Although the test deploys the test-plugin, the output also indicates that this plugin was created 4 hours earlier.

Since I cannot reproduce the problem with this test locally, and performing the test on a fresh environment succeeds, I think the problem must lay outside of my commits, and has been present for a while. 

@felipe: this is integration test 5. I see that integration test 3 is empty. Could we move this test to the slot of int-3 and see if this fixes it? If so, we can conclude that int-4 is messing up the platform somehow.
